### ISSUE TITLE
Fix `declaration-block-no-redundant-longhand-properties` false negatives for `inset` shorthand

### ISSUE THREAD

**Issue by Mouvedia** [stylelint/stylelint#6613]
### Demand

```
top: 0;
right: 0;
bottom: 0;
left: 0;
```
should get fixed to
```
inset: 0;
```

i.e. add `inset` property to [this list](https://github.com/stylelint/stylelint/blob/autofix-declaration-block-no-redundant-longhand-properties/lib/rules/declaration-block-no-redundant-longhand-properties/README.md?plain=1#L27)

prompted by https://github.com/stylelint/stylelint/pull/6580#discussion_r1096573466

**Comment by DeRaowl** [comment]
Hi @Mouvedia , I have gone through the issue and can I work on this one?

**Comment by Mouvedia** [comment]
@DeRaowl normally you should wait for the `status: ready to implement` to be added to the issue.
@jeddy3 can I add it?

one caveat though: if you start before #6580 merge, you will have to refactor your code

**Comment by DeRaowl** [comment]
Ack, will wait for the status to be updated. Thank you.

**Comment by Mouvedia** [comment]
@DeRaowl Meanwhile you can read about [how to fix a bug in a rule](https://stylelint.io/developer-guide/rules#fix-a-bug-in-a-rule) in the Developer guide and check the other [good first issues that are ready to be implemented](https://github.com/stylelint/stylelint/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22+label%3A%22status%3A+ready+to+implement%22).

**Comment by jeddy3** [comment]
@Mouvedia Thanks for creating the issue.

@DeRaowl Thanks for offering to work on it.

> one caveat though: if you start before https://github.com/stylelint/stylelint/pull/6580 merge, you will have to refactor your code

Let's wait until https://github.com/stylelint/stylelint/pull/6580 is merged before starting

**Comment by DeRaowl** [comment]
@jeddy3 can I pick this issue?

**Comment by ybiquitous** [comment]
@DeRaowl Yeah, of course!

**Comment by hail2u** [comment]
I think the rule is also false negative for `(border|margin|padding)-(block|inline)`. For example:

```css
margin-block-end: 0;
margin-block-start: 0;
```

should be:

```css
margin-block: 0;
```

**Comment by Mouvedia** [comment]
It's been more than 2 weeks, it's available again.

**Comment by rayrw** [comment]
Hi, I'd like to pick this one if it's okay.
